### PR TITLE
RDCC-1440, Master: removing installCharts() from Jenkinsfiles

### PR DESCRIPTION
**Pull request by AdilOoze**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-1440

### Change description ###

removing installCharts() from Jenkinsfiles

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: config/owasp/suppressions.xml [hunk 1]
```diff
@@ -1605,4 +1605,105 @@
...
+    <suppress>
+        <notes><![CDATA[
+   file name: dom4j-2.1.1.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.dom4j/dom4j@.*$</packageUrl>
+        <cve>CVE-2020-10683</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: ant-1.9.6.jar
```
Comment (abhijitdiwan) [inline config/owasp/suppressions.xml:1669]: we can replace this ant with <suppress>
     <notes><![CDATA[
file name: ant*.jar
]]></notes>
     <filePath regex="true">.*\**\.jar</filePath>
     <cpe>cpe:/a:apache:ant</cpe>
</suppress>

Comment (AdilOoze) [reply config/owasp/suppressions.xml]: Addressed in this commit: https://github.com/hmcts/rd-profile-sync/pull/239/commits/dd45338b9a59bdc7dcebad76be65a4c84f5836d8

File: config/owasp/suppressions.xml [hunk 2]
```diff
@@ -1605,4 +1605,84 @@
...
     </suppress>
+    <suppress>
+        <notes><![CDATA[
+   file name: spring-security-config-5.2.1.RELEASE.jar
+   ]]></notes>
+        <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-config@.*$</packageUrl>
+        <cpe>cpe:/a:pivotal_software:spring_security</cpe>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
```
Comment (abhijitdiwan) [inline config/owasp/suppressions.xml:1616]: can't we make compact/lesser lines for spring security? Seems duplicated lines

Comment (AdilOoze) [reply config/owasp/suppressions.xml:1616]: Addressed removal of duplicate CVEs in this commit: https://github.com/hmcts/rd-profile-sync/pull/239/commits/cd691c20bcc0d939adddabc7800b9c91edc70303

Comment (AdilOoze) [reply config/owasp/suppressions.xml:1616]: Addressed removal of duplicate CPEs in this commit: https://github.com/hmcts/rd-profile-sync/pull/239/commits/21e64b3a8fef3f59d9d346d7efb65f50f1e7dd6b

**Pull request merged**